client/src/components/Tabs/Tabs.tsx

```diff
@@ -77,24 +73,24 @@ const ContentTabs: FC<Props> = ({ topic, tabs = ['Lesson', 'Code', 'Videos'] })
         </TabPanel>
       ))}
     </Box>
-  // <Grid container>
-  //     <Grid item container className="Tab-Group" direction="row" spacing={0}>
-  //         {tabs.map((label, idx) => (
-  //             <Grid item key={label}>
-  //                 <button
-  //                     type="button"
-  //                     onClick={() => setTabIndex(idx)}
-  //                     className={tabIndex === idx ? 'Tab-Selected' : 'Tab'}
-  //                 >
-  //                     {label}
-  //                 </button>
-  //             </Grid>
-  //         ))}
-  //     </Grid>
-  //     <Grid item style={{ margin: '24px 12px 16px 12px', width: '100%' }}>
-  //
-  //     </Grid>
-  // </Grid>
+    // <Grid container>
+    //     <Grid item container className="Tab-Group" direction="row" spacing={0}>
+    //         {tabs.map((label,  i) => (
+    //             <Grid item key={label}>
+    //                 <button
+    //                     type="button"
+    //                     onClick={() => setTabIndex( i)}
+    //                     className={tabIndex ===  i ? 'Tab-Selected' : 'Tab'}
+    //                 >
+    //                     {label}
+    //                 </button>
+    //             </Grid>
+    //         ))}
+    //     </Grid>
+    //     <Grid item style={{ margin: '24px 12px 16px 12px', width: '100%' }}>
+    //
+    //     </Grid>
+    // </Grid>
```

Comment:
Why the commented out code?

Reply:
My IDE linter wasn't sourcing from the same rules it looks like. Reverted this

Reply:
I meant what's this for?

Reply:
Oh good point, this was actually an older component that Alex wrote I think. This doesn't need to be here anymore